snap/local/enb.conf

```diff
@@ -0,0 +1,429 @@
+#####################################################################
+[rf]
+#dl_earfcn = 3350
+tx_gain = 80
+rx_gain = 40
+
+#device_name = auto
+
+# For best performance in 2x2 MIMO and >= 15 MHz use the following device_args settings:
+#     USRP B210: num_recv_frames=64,num_send_frames=64
+#     And for 75 PRBs, also append ",master_clock_rate=15.36e6" to the device args
+
+# For best performance when BW<5 MHz (25 PRB), use the following device_args settings:
+#     USRP B210: send_frame_size=512,recv_frame_size=512
+
+#device_args = auto
+#time_adv_nsamples = auto
+
+# Example for ZMQ-based operation with TCP transport for I/Q samples
+device_name = zmq
+device_args = fail_on_disconnect=true,tx_port=tcp://*:2000,rx_port=tcp://localhost:2001,id=enb,base_srate=23.04e6
+
+#####################################################################
+# Packet capture configuration
+#
+# MAC-layer packets are captured to a file in the compact format which can
+# be decoded by Wireshark. For decoding, use the UDP dissector and the UDP 
+# heuristic dissection. Edit the preferences (Edit > Preferences > 
+# Protocols > DLT_USER) for DLT_USER to add an entry for DLT=149 with 
+# Protocol=udp. Further, enable the heuristic dissection in UDP under:
+# Analyze > Enabled Protocols > MAC-LTE > mac_lte_udp and MAC-NR > mac_nr_udp
+# For more information see: https://wiki.wireshark.org/MAC-LTE
+# Configuring this Wireshark preferences is needed for decoding the MAC PCAP
+# files as well as for the live network capture option.
+#
+# Please note that this setting will by default only capture MAC
+# frames on dedicated channels, and not SIB.  You have to build with
+# WRITE_SIB_PCAP enabled in srsenb/src/stack/mac/mac.cc if you want
+# SIB to be part of the MAC pcap file.
+#
+# S1AP Packets are captured to a file in the compact format which can
+# be decoded by the Wireshark s1ap dissector with DLT 150. 
+# To use the dissector, edit the preferences for DLT_USER to 
+# add an entry with DLT=150, Payload Protocol=s1ap.
+#
+# enable:        Enable MAC layer packet captures (true/false)
+# filename:      File path to use for LTE MAC packet captures
+# nr_filename:   File path to use for NR MAC packet captures
+# s1ap_enable:   Enable or disable the PCAP.
+# s1ap_filename: File name where to save the PCAP.
+#
+# mac_net_enable: Enable MAC layer packet captures sent over the network (true/false default: false)
+# bind_ip: Bind IP address for MAC network trace (default: "0.0.0.0")
+# bind_port: Bind port for MAC network trace (default: 5687)
+# client_ip: Client IP address for MAC network trace (default: "127.0.0.1")
+# client_port Client IP address for MAC network trace (default: 5847)
+#####################################################################
+[pcap]
+#enable = false
+#filename = /var/snap/srsran/current/enb_mac.pcap
+#nr_filename = /var/snap/srsran/current/enb_mac_nr.pcap
+#s1ap_enable = false
+#s1ap_filename = /var/snap/srsran/current/enb_s1ap.pcap
```

Comment:
I think the change will not take effect if the lines stay commented out. I would uncomment all the `filename` lines where we change a default.

Reply:
Fixed